src/amo/api/index.js

```diff
@@ -41,6 +41,25 @@ export function submitReview({
     });
 }
 
+export function getAddonReviews({ addonSlug } = {}) {
+  return new Promise((resolve) => {
+    if (!addonSlug) {
+      throw new Error('addonSlug cannot be falsey');
+    }
+    resolve(callApi({
+      endpoint: `addons/addon/${addonSlug}/reviews`,
+      method: 'GET',
+    }));
+  })
+    .then((response) => {
+      // TODO: implement paging through response.next
+      if (response.next) {
+        throw new Error('paging is not yet implemented');
```

Comment:
Isn't not having a pager enough though? When testing this against dev it was a bit annoying to have to find an add-on with few enough reviews to fit on one page.